test/unit/test_session.js

```diff
@@ -22,7 +22,7 @@ var test = {
     connect: { options: { containerId: 'test' } },
     senderLink: { attach: { name: 'sender' } },
     receiverLink: { attach: { name: 'receiver' } }
-  }, DefaultPolicy)
+  }, DefaultPolicy),
```

Comment:
>, [](start = 19, length = 1)

typo?

Reply:
probably had something else and took it out. good catch